packages/redeem-failed-retryable/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "redeem-failed-retryable",
+  "license": "Apache-2.0",
+  "version": "1.0.0",
+  "scripts": {
+    "build": "hardhat compile"
```

Comment:
maybe add the script here?

Reply:
It's not a script though, a task was created (instead of using script) in this case to receive user argument (L1 tx hash)